virttest/qemu_virtio_port.py

```diff
@@ -98,7 +99,11 @@ def open(self):     # @ReservedAssignment
         attempt = 11
         while attempt > 0:
             try:
-                self.sock = socket.socket(socket.AF_UNIX,
+                if self.port_type == 'unix_socket':
+                    flag = socket.AF_UNIX
+                elif self.port_type in ('tcp_socket', 'udp'):
+                    flag = socket.AF_INET
+                self.sock = socket.socket(flag,
```

Comment:
what's code for UDP socket?

socket.socket(flag, socket.SOCK_DGRAM)

Reply:
@xutian Did not notice that before, only test tcp, thanks for reminder, will check it.

Reply:
@xutian Updated, thanks.